plugin/src/leiningen/cljs_lambda/aws.clj

```diff
@@ -39,9 +44,9 @@
                               "]"])}))))
 
 (defn ->cli-args [m & [positional {:keys [preserve-names?]}]]
-  (let [m    (cond-> (merge (meta-config) (update-vpc-value m))
+  (let [m    (cond-> (merge (meta-config) (-> m (update-vpc-value) (update-dead-letter-value)))
                (not preserve-names?)
-               (set/rename-keys {:name :function-name :vpc :vpc-config}))
```

Comment:
Sounds good. Away from computer over the weekend, but already started work on env support in another branch where I can include these changes. Another transformation of env variables, needs to be refactored little bit to make it nicer.